src/anomalib/models/components/base/anomaly_module.py

```diff
@@ -220,30 +258,12 @@ def input_size(self) -> tuple[int, int] | None:
         The effective input size is the size of the input tensor after the transform has been applied. If the transform
         is not set, or if the transform does not change the shape of the input tensor, this method will return None.
         """
-        transform = self.transform or self.configure_transforms()
+        transform = self.pre_processor.predict_transform
```

Comment:
If we allow the user to disable pre_processing,`self.pre_processor` could be `None`, which would cause an error here.

Reply:
Addressed in 401fbaa15374aa2ea00c5c29da72ec0548296b45